test/internalisation/test-totalSupply-definition.kore.report

```diff
@@ -1980,7 +1980,7 @@ Symbols: 1720
   - inj
   - kseq
 Axioms: 352
-Axioms preserving definedness: 295
+Axioms preserving definedness: 292
```

Comment:
This was caused by an imprecise analysis in `filterTermSymbols` (the concat symbol was checked in cases of singleton sets where it would not be required).